src/main/resources/db/seed/local+dev+test/cas2_application_data/data_A1234AI.json

```diff
@@ -13,8 +13,23 @@
       "consentDate-day": "1"
     }
   },
+  "hdc-licence-dates": {
+    "hdc-licence-dates": {
```

Comment:
just a thought - do we need this double nesting here? `hdc-licence-dates` inside `hdc-licence-dates`?

Reply:
good spot but no, the outer label refers to the task and the inner label to the page. Because it's a task with a single page, the two may as well be the same